cmd/sonobuoy/app/pluginList.go

```diff
@@ -71,19 +74,53 @@ func (p *pluginList) Set(str string) error {
 	case pluginSystemdLogs:
 		p.DynamicPlugins = append(p.DynamicPlugins, str)
 	default:
-		b, err := ioutil.ReadFile(str)
+		finfo, err := os.Stat(str)
```

Comment:
Since I have to stat to check if it is a directory or not, I had to have a new possible error message and so some tests needed updated for this reason as well. Tiny change but impacts a handful of tests.